integration-tests/modules/__tests__/cart/store/cart.completion.ts

```diff
@@ -226,6 +226,13 @@ medusaIntegrationTestRunner({
             },
           })
 
+          completeCartWorkflow.hooks.beforePaymentAuthorization((input) => {
+            expect(input.input.id).toBeDefined()
+          })
+          completeCartWorkflow.hooks.orderCreated((input) => {
+            expect(input.order_id).toBeDefined()
+          })
+
```

Comment:
Can we test if these methods are undefined when `orderId` does not exist?

Reply:
@thetutlage yeah I wanted to discuss in Slack first, but they are defined, and they execute even if we don't enter in the when with data missing (i.e. `input.input.id` is defined, but `input.order_id` isn't) .